frontend/src/components/common/Table.stories.tsx

```diff
@@ -0,0 +1,453 @@
+import { Box, Typography } from '@mui/material';
```

Comment:
These are nice.

I was wondering if you could do some accessibility testing on it? (if you have already, could please let me know what you covered?).

Reply:
Keyboard navigation works fine, there are aria-labels set on everything important. One interesting thing about table a11y is how it works with screen readers. When screen reader is enabled it provides new ways to navigate table, like arrow keys to go to cells within a table. Screen reader needs to detect semantic elements for table (table,th,tr,rd,etc). There's an [issue](https://github.com/w3c/csswg-drafts/issues/3040) with display: contents (that we use) that kind of breaks a11y tree but I've tested it with NVDA on Windows and it seems to be working great, can jump to table landmark and navigate the cells.

Reply:
Nice one, thanks.

We also have react-axe making warnings in the console if there are any runtime issues it detects. I wonder if you've checked if any new warnings were printed in the console from axe?

Did you also check the lighthouse accessibility audit in the dev tools?

Reply:
good idea, just checked and neither axe nor lighthouse report any issues with the table